### PR TITLE
Add FHIR-overview page

### PR DESCRIPTION
**Pull request by isuruh15**

### CODE REVIEW

File: en/docs/fhir/guides/overview-of-fhir.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+2. **Interoperability**: FHIR is designed to facilitate interoperability between different healthcare systems. It supports both simple data exchange (like patient demographics) and complex workflows (like care coordination).
+3. **API-based**: It's designed to work with modern web technologies, using RESTful APIs for data exchange. Which means data can be easily retrieved and updated over the web using HTTP requests.
+4. **Flexibility and Extensibility**: FHIR allows for customization to meet specific requirements while maintaining a core set of common elements.
+5. **Support for Multiple Formats**: FHIR supports data representation in JSON, XML, and RDF formats, making it compatible with various systems and easier to work with in different programming environments.
+6. **Security**: FHIR incorporates standard security protocols, including OAuth2 and SMART on FHIR, to ensure that patient data is securely accessed and shared.
+
+WSO2 Healthcare provides a comprehensive set of products/tools to support the FHIR-based digital transformation and compliance needs of any payer, provider, or related business.
+
+???+ note
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/overview-of-fhir.md]: Can you put this as a reusable component in md,

Add a file like following

!!! note
This is a reusable note. You can include this note in any document by referencing this file.

Refer in the actual guide

{!includes/note.md!}

Comment (isuruh15) [reply en/docs/fhir/guides/overview-of-fhir.md]: fixed in 
558e0e6

File: en/docs/fhir/guides/overview-of-fhir.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+2. **Interoperability**: FHIR is designed to facilitate interoperability between different healthcare systems. It supports both simple data exchange (like patient demographics) and complex workflows (like care coordination).
+3. **API-based**: It's designed to work with modern web technologies, using RESTful APIs for data exchange. Which means data can be easily retrieved and updated over the web using HTTP requests.
+4. **Flexibility and Extensibility**: FHIR allows for customization to meet specific requirements while maintaining a core set of common elements.
+5. **Support for Multiple Formats**: FHIR supports data representation in JSON, XML, and RDF formats, making it compatible with various systems and easier to work with in different programming environments.
+6. **Security**: FHIR incorporates standard security protocols, including OAuth2 and SMART on FHIR, to ensure that patient data is securely accessed and shared.
+
+WSO2 Healthcare provides a comprehensive set of products/tools to support the FHIR-based digital transformation and compliance needs of any payer, provider, or related business.
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/overview-of-fhir.md]:
```suggestion
WSO2 Healthcare provides a comprehensive set of features/tools to support the FHIR-based digital transformation and compliance needs of any payer, provider, or related business.
```

File: en/docs/fhir/guides/overview-of-fhir.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+{!includes/bal-mi-note.md!}
+
+
+=== "Ballerina"
+<Ballering doc content>
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/overview-of-fhir.md]: <Ballering doc content> Not needed right? same with <MI doc content>

File: en/mkdocs.yml [hunk 4]
```diff
@@ -42,7 +42,6 @@ theme:
   feature:
     tabs: true
   language: en
-
 # Navigation
```
Comment (sameeragunarathne) [inline en/mkdocs.yml:45]: might need to add the following plugin to the plugins section:

```
plugins:
  - include-markdown
```

Comment (isuruh15) [reply en/mkdocs.yml:45]: worked without the plugin locally. anyway added the plugin in [69b7592](https://github.com/wso2/open-healthcare-docs/pull/28/commits/69b75929cc69eb952cd42365740b98a282b368d4)

File: en/docs/fhir/guides/overview-of-fhir.md (hunk 3 above)
Comment (isuruh15) [reply en/docs/fhir/guides/overview-of-fhir.md]: changed in [69b7592](https://github.com/wso2/open-healthcare-docs/pull/28/commits/69b75929cc69eb952cd42365740b98a282b368d4)

File: en/docs/fhir/guides/overview-of-fhir.md (hunk 2 above)
Comment (isuruh15) [reply en/docs/fhir/guides/overview-of-fhir.md]: added in [69b7592](https://github.com/wso2/open-healthcare-docs/pull/28/commits/69b75929cc69eb952cd42365740b98a282b368d4)

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**